apps/hash-frontend/src/components/grid/utils/sorting.ts

```diff
@@ -43,11 +20,12 @@ export const defaultSortRows = <T extends Row>(
   const clone = [...rows] as T[];
   return clone.sort((row1, row2) => {
     // we sort only by alphabetical order for now
-    const key1 = String(row1[sort.key]);
-    const key2 = String(row2[sort.key]);
+    const key1 = String(row1[sort.columnKey]);
+    const key2 = String(row2[sort.columnKey]);
+
     let comparison = key1.localeCompare(key2);
 
-    if (sort.dir === "desc") {
+    if (sort.direction === "desc") {
       // reverse if descending
       comparison = -comparison;
     }
```

Comment:
I think it would be nice to be able to sort on multiple columns... one way of moving towards this would be to store the previously-sorted column and use it as a tie-breaker if the rows are equal for the active sort column.